templates/zerver/config_error.html

```diff
@@ -80,13 +80,13 @@ <h1 class="lead">Configuration error</h1>
                         {% endif %}
                     {% endif %}
 
-                    {% if remoteuser_error_backend_disabled %}
+                    {% if social_backend_name == "remoteuser_error_backend_disabled" %}
                     <p>
                         Authentication via the REMOTE_USER header is
                         disabled in `/etc/zulip/settings.py`.
                     </p>
                     {% endif %}
-                    {% if remoteuser_error_remote_user_header_missing %}
+                    {% if social_backend_name == "remoteuser_error_remote_user_header_missing" %}
```

Comment:
I think we might want these to be `if error_name ==`, and store these special errors in a separate field.